### PR TITLE
[Improve] Localization streampark-usercase-joyme

### PR DESCRIPTION
**Pull request by VampireAchao**
see: https://github.com/apache/incubator-streampark/issues/3218

cc: @caicancai @saLeox

### CODE REVIEW

File: blog/6-streampark-usercase-joyme.md [hunk 1]
```diff
@@ -1,46 +1,46 @@
 ---
 slug: streampark-usercase-joyme
-title: StreamPark 在 Joyme 的生产实践
-tags: [StreamPark, 生产实践, FlinkSQL]
+title: StreamPark in Joyme's Production Practice
```
Comment (saLeox) [inline blog/6-streampark-usercase-joyme.md:3]: title: StreamPark's Production Practic in Joyme

File: blog/6-streampark-usercase-joyme.md [hunk 2]
```diff
@@ -69,93 +69,93 @@ CREATE TABLE sink_table (
...
 );
 
--- 代码逻辑过
+-- Code logic pass
 INSERT INTO sink_table
 SELECT  Data.uid  FROM source_table;
 ```
 
-### **2. 添加依赖**
+### **2. Dependency Addition**
```
Comment (saLeox) [inline blog/6-streampark-usercase-joyme.md:77]: ### **2. Add Dependency**

File: blog/6-streampark-usercase-joyme.md [hunk 3]
```diff
@@ -69,93 +69,93 @@ CREATE TABLE sink_table (
...
 
-关于依赖这块是 StreamPark 里特有的，在 StreamPark 中创新型的将一个完整的 Flink Sql 任务拆分成两部分组成: Sql 和 依赖, Sql 很好理解不多啰嗦, 依赖是 Sql 里需要用到的一些 Connector 的 Jar, 如 Sql 里用到了 Kafka 和 MySQL 的 Connector, 那就需要引入这两个 Connector 的依赖, 在 StreamPark 中添加依赖两种方式，一种是基于标准的 Maven pom 坐标方式，另一种是从本地上传需要的 Jar 。这两种也可以混着用，按需添加，点击应用即可， 在提交作业的时候就会自动加载这些依赖。
+In terms of dependencies, StreamPark has a unique approach. A complete Flink SQL job is innovatively split into two components within StreamPark: the SQL and the dependencies. The SQL part is self-explanatory and requires no further explanation, but the dependencies are the various Connector JARs needed within the SQL, such as Kafka and MySQL Connectors. If these are used within the SQL, then these Connector dependencies must be introduced. In StreamPark, there are two ways to add dependencies: one is based on the standard Maven pom coordinates, and the other is by uploading the required Jars from a local source. These two methods can also be mixed and used as needed; simply apply, and these dependencies will be automatically loaded when the job is submitted.
```
Comment (saLeox) [inline blog/6-streampark-usercase-joyme.md:79]: In terms of dependencies, it's an unique feature to Streampark.
The SQL part is easy to understand and requires no further explanation

File: blog/6-streampark-usercase-joyme.md [hunk 4]
```diff
@@ -69,93 +69,93 @@ CREATE TABLE sink_table (
...
 ![](/blog/joyme/checkpoint_configuration.png)
 
-### **4. 动态参数设置**
+### **4. Dynamic Parameter Settings**
 
-由于我们的模式部署是 on Yarn，在动态选项配置里配置了 Yarn 的队列名称。也有一些配置了开启增量的 Checkpoint 选项和状态过期时间，基本的这些参数都可以从 Flink 的官网去查询到。之前有一些作业确实经常出现内存溢出的问题，加上增量参数和过期参数以后，作业的运行情况好多了。还有就是 Flink Sql 作业设计到状态这种比较大和逻辑复杂的情况下，我个人感觉还是用 Streaming 代码来实现比较好控制一些。
+Since our deployment mode is on Yarn, we configured the name of the Yarn queue in the dynamic options. Some configurations have also been set to enable incremental checkpoints and state TTL (time-to-live), all of which can be found on Flink's official website. Before, some jobs frequently encountered out-of-memory issues. After adding the incremental parameter and TTL, the job operation improved significantly. Also, in cases where the Flink SQL job involves larger states and complex logic, I personally feel that it's better to implement them through streaming code for more control.
 
-- -Dyarn.application.queue= yarn队列名称
+- -Dyarn.application.queue= [Yarn queue name]
```
Comment (saLeox) [inline blog/6-streampark-usercase-joyme.md:95]: No []

File: blog/6-streampark-usercase-joyme.md [hunk 5]
```diff
@@ -69,93 +69,93 @@ CREATE TABLE sink_table (
...
 - -Dstate.backend.incremental=true
-- -Dtable.exec.state.ttl=过期时间
+- -Dtable.exec.state.ttl= [expiration time]
```
Comment (saLeox) [inline blog/6-streampark-usercase-joyme.md:97]: No []

File: blog/6-streampark-usercase-joyme.md [hunk 6]
```diff
@@ -69,93 +69,93 @@ CREATE TABLE sink_table (
...
 
-完成配置以后提交，然后在 application 界面进行部署。
+After completing the configuration, submit it, and then deploy it from the application interface.
```
Comment (saLeox) [inline blog/6-streampark-usercase-joyme.md:99]: After completing the configuration, submit & deploy it from the application interface.

File: blog/6-streampark-usercase-joyme.md [hunk 7]
```diff
@@ -69,93 +69,93 @@ CREATE TABLE sink_table (
...
 
-新建任务，选择 Custom code ，选择 Flink 版本，选择项目以及模块 Jar 包，选择开发的应用模式为 Apache Flink (标准的 Flink 程序)，程序主函数入口类，任务的名称。
+To create a new task, select Custom code, choose the Flink version, select the project and the module Jar package, and choose the development application mode as Apache Flink (standard Flink program), program main function entry class, and the task's name.
 
 ![](/blog/joyme/add_projectconfiguration.png)
 
-以及任务的并行度，监控的方式等，内存大小根据任务需要进行配置。Program Args 程序的参数则根据程序需要自行定义入口参数,比如：我们统一启动类是 StartJobApp，那么启动作业就需要传入作业的 Full name 告诉启动类要去找哪个类来启动此次任务，也就是一个反射机制，作业配置完成以后同样也是 Submit 提交，然后在 application 界面部署任务。
+As well as the task’s parallelism, monitoring method, etc., memory size should be configured based on the needs of the task. Program Args, the program parameters, are defined according to the program's needs. For example: If our unified startup class is StartJobApp, to start a job, it's necessary to pass the full name of the job, informing the startup class which class to find to launch the task—essentially, a reflection mechanism. After the job configuration is complete, it is also submitted with Submit and then deployed from the application interface.
```
Comment (saLeox) [inline blog/6-streampark-usercase-joyme.md:117]: informing the startup class which class to find to launch the task—essentially, which is a reflection mechanism.

File: blog/6-streampark-usercase-joyme.md [hunk 8]
```diff
@@ -69,93 +69,93 @@ CREATE TABLE sink_table (
...
 
-以上就是 StreamPark 在乐我无限的全部分享内容，感谢大家看到这里。非常感谢 StreamPark 提供给我们这么优秀的产品，这就是做的利他人之事。从1.0 到 1.2.1 平时遇到那些bug都会被即时的修复，每一个issue都被认真对待。目前我们还是 onyarn的部署模式，重启yarn还是会导致作业的lost状态，重启yarn也不是天天都干的事，关于这个社区也会尽早的会去修复这个问题。相信 StreamPark 会越来越好，未来可期。
+That is all the content shared by StreamPark at LeWo Unlimited. Thank you for reading this far. We are very grateful for such an excellent product provided by StreamPark, which is a true act of benefiting others. From version 1.0 to 1.2.1, the bugs encountered are promptly fixed, and every issue is taken seriously. We are still using the onyarn deployment mode. Restarting yarn will still cause jobs to be lost, but restarting yarn is not something we do every day. The community will also look to fix this problem as soon as possible. I believe that StreamPark will get better and better, with a promising future ahead.
```
Comment (saLeox) [inline blog/6-streampark-usercase-joyme.md:161]: That is all the content shared by StreamPark at Joyme. Thank you for reading this article.
We are still using the on yarn deployment mode.

**Comment by VampireAchao** [comment]
Thk U for review,I'll finish them ASAP.

**Comment by saLeox** [comment]
LGTM

**Review by wolfboys — APPROVED** [review]

**Pull request merged**